canonical-websites/snapcraft.io
# Move back api module to s.io

# Summary

Fixes https://github.com/canonicalltd/snap-squad/issues/587
After discussion we decided to move back the project `canonicalwebteam.snapstoreapi` to `snapcraft.io` because the API continue to evolve and no other project are requiring for now this external module.

This PR is:
 - Move back the code from the module to s.io
 - Put all `api` dependecy in `webapp.api` (for example the files: `cache`, `exceptions`, `blog api`)

# QA

- `./run`
- Navigate through the site and make sure that the APIs calls are well made
